packages/blocks/src/store/reducer.js

```diff
@@ -393,6 +393,13 @@ function getMergedUsesContext( existingUsesContext = [], newUsesContext = [] ) {
 export function blockBindingsSources( state = {}, action ) {
 	switch ( action.type ) {
 		case 'ADD_BLOCK_BINDINGS_SOURCE':
+			// Only open this API in Gutenberg and for `core/post-meta` for the moment.
+			let getFieldsList;
+			if ( globalThis.IS_GUTENBERG_PLUGIN ) {
+				getFieldsList = action.getFieldsList;
+			} else if ( action.name === 'core/post-meta' ) {
+				getFieldsList = action.getFieldsList;
+			}
```

Comment:
@gziolo Could you confirm this is the proper way to enable the API for Gutenberg but keep it private for WordPress 6.7?

PD: I had to split the conditional this way because the linter was complaining that `globalThis.IS_GUTENBERG_PLUGIN` can't be used among other checks.

Reply:
Yes, that looks right. In effect, all custom sources can test `getFieldsList` with the Gutenberg plugin, but only `core/post-meta` is handled with WP core.